### ISSUE TITLE
Event subscription not getting created from first block

### ISSUE THREAD

**Issue by nguyer** [hyperledger/firefly-fabconnect#57]
It appears that fabconnect isn't subscribing to blockchain events from the beginning of the chain. It looks like it's not subscribing to blockchain events until after the first transaction is complete, and by then it's already missed an event.

This causes the very first transaction that FireFly submits to end up timing out. When running fabric with the FireFly CLI, this causes the first registration request to timeout, which means the CLI will sit there for 2 minutes waiting for the transaction before trying again.

Steps to reproduce:

Using the FireFly CLI create a new fabric stack and start it:
```
ff init fabric 2 -b fabric
ff start fabric
```
You'll see the CLI output like this:
```
registering FireFly identities
registering org and node for member 0
Post "http://127.0.0.1:5000/api/v1/network/organizations/self?confirm=true": EOF - retrying request...http://127.0.0.1:5000/api/v1/network/organizations/self?confirm=true returned 408: {"error":"FF10260: The request with id 'dcdd12c7-cf51-45e3-83c1-89faab8de669' timed out after 119,975.64ms"}
 - retrying request...registering org and node for member 1
initializing token providers


Web UI for member '0': http://127.0.0.1:5000/ui
Web UI for member '1': http://127.0.0.1:5001/ui

To see logs for your stack run:

ff logs fabric
```

Here is a log from fabconnect that shows the first transaction submitted, and where it subscribes to blockchain events (afterward, at block 4):
```
[2021-10-22T14:22:18.744Z]  INFO --> POST /transactions?fly-chaincode=firefly&fly-channel=firefly&fly-id=c872081c-17fa-433e-a9aa-079ab93aa5f8&fly-signer=0xf5975feaec8a3cdd5ecf22a5f6cb45de943b549f&fly-sync=false
[2021-10-22T14:22:18.744Z]  INFO Request handler accepted message. MsgID: c872081c-17fa-433e-a9aa-079ab93aa5f8 Type: SendTransaction
[2021-10-22T14:22:18.744Z]  INFO In-flight 1000000 added. signer=0xf5975feaec8a3cdd5ecf22a5f6cb45de943b549f before=0
[2021-10-22T14:22:18.745Z]  INFO <-- POST /transactions?fly-chaincode=firefly&fly-channel=firefly&fly-id=c872081c-17fa-433e-a9aa-079ab93aa5f8&fly-signer=0xf5975feaec8a3cdd5ecf22a5f6cb45de943b549f&fly-sync=false [202]:
{"sent":true,"id":"c872081c-17fa-433e-a9aa-079ab93aa5f8"}
[2021-10-22T14:22:20.775Z]  INFO TX: Sent OK [2.03s]
[2021-10-22T14:22:20.876Z]  INFO Obtained receipt after 100.30ms - average time to receipt: 100.30ms
[2021-10-22T14:22:20.876Z]  INFO Receipt for 8f2ca06a11ed59b84d75ace291636048b3690f85a9c0da6ba6ec61dd22364d69 obtained after 0.10s Success=true
[2021-10-22T14:22:20.876Z]  INFO Received reply message. requestId='c872081c-17fa-433e-a9aa-079ab93aa5f8' reqOffset='' type='TransactionSuccess':
[2021-10-22T14:22:20.877Z]  INFO c872081c-17fa-433e-a9aa-079ab93aa5f8: Inserted receipt into receipt store
[2021-10-22T14:22:20.877Z]  INFO In-flight 1000000 complete. signer=0xf5975feaec8a3cdd5ecf22a5f6cb45de943b549f sub=true before=1 after=0
[2021-10-22T14:22:26.627Z]  INFO sb-1c610c7d-abf1-452b-7e5c-0192fa0a334a: initial block height for subscription (latest block): 4
[2021-10-22T14:22:26.629Z]  INFO Subscribed to events in channel firefly chaincode firefly from block 4 (0 means newest)
[2021-10-22T14:22:26.629Z]  INFO sb-1c610c7d-abf1-452b-7e5c-0192fa0a334a: created filter from block 4: {BlockType:tx ChaincodeId:firefly EventFilter:BatchPin Filter: ToBlock:0}
```

**Comment by awrichar** [comment]
There are two layers to this issue:
1. FireFly is not passing `fromBlock: "0"` when it [creates the subscription to fabconnect](https://github.com/hyperledger/firefly/blob/ee167998727f4dceb4f56baa4f7614bfabb8ebfd/internal/blockchain/fabric/eventstream.go#L112)
2. fabconnect doesn't appear to respect `fromBlock: "0"` even if it is passed

I tried a manual test by sending some transactions to fabconnect, then later creating an event stream and a subscription with `fromBlock: "0"`. I observed that none of the previous events were delivered, but new events were.
